healpy/visufunc.py

```diff
@@ -387,6 +388,8 @@ def gnomview(
       If True: do not add resolution info text. Default=False
     return_projected_map : bool
       if True returns the projected map in a 2d numpy array
+    no_plot : bool
```

Comment:
Could you make it more consistent and use
`no_plot : bool, optional`?
(and please also add that to the `return_projected_map` keyword)